permissions_engine/fetch_keys.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import json
+issuers = ["http://localhost:8080/auth/realms/mockrealm"]
+
+data = dict()
+data["keys"] = dict()
+data["keys"]["https://oidc:8443/auth/realms/mockrealm"] = requests.get("http://localhost:8080/auth/realms/mockrealm/protocol/openid-connect/certs").text
+
+
+
```

Comment:
We should make this code above look more like the code below; is there a reason why that's commented out?

Reply:
the issuer is https://oidc:8443/auth/realms/mockrealm/ instead of the host that we visited to get the certificate http://localhost:8080/auth/realms/mockrealm/.
I am not sure how to proceed